src/findCollisions.js

```diff
@@ -0,0 +1,27 @@
+
+module.exports = function findCollisions (timetable, constraint) {
```

Comment:
Also, the convention is to use `() =>` instead of `function` when possible (which is always, because nobody wants brittle code which relies on `this`)